hbase-assembly/src/main/assembly/client.xml

```diff
@@ -162,15 +165,6 @@
         <include>io.opentelemetry.javaagent:*</include>
       </includes>
     </dependencySet>
-    <!-- Adds junit libs to lib/test -->
```

Comment:
We need to somehow make sure that the test jars are available when starting HBase from the source directory.

Reply:
Would be handled with comment https://github.com/apache/hbase/pull/6184/files#r1799303394

Reply:
I have deleted these as we need them only in dev mode and for that we already are handling it via  cached_classpath